File: INECVotingApp/e-voters_app/Candidates.py
class Candidates:
    def __init__(self, name, candidate_id, candidate_age, pincode, vote_count):
        if candidate_age >= 40:
            self.name = name
            self.candidate_id = candidate_id
            self.candidate_age = candidate_age
            self.pincode = pincode
            self.vote_count = vote_count
        else:
            raise ValueError("Candidate must be at least 40 years old")

    def get_name(self):
        return self.name

    def get_candidate_age(self):
        return self.candidate_age

    def get_candidate_id(self):
        return self.candidate_id

    def get_vote_count(self):
        return self.vote_count

    def increment_vote(self):
        self.vote_count += 1

    def validate_candidate_pincode(self, pincode):
        if pincode != self.pincode:
            raise ValueError("Invalid Pincode")

File: INECVotingApp/Candidates.java
package INECVotingApp;

public class Candidates {
    private String name;
    private int candidateID;
    private int candidateAge;
    private String pincode;
    private int voteCount;

    public Candidates(String name, int candidateID, int candidateAge, String pincode, int voteCount) {
        if (candidateAge >= 40) {
            this.name = name;
            this.candidateID = candidateID;
            this.candidateAge = candidateAge;
            this.pincode = pincode;
            this.voteCount = voteCount;
        } else {
            throw new IllegalArgumentException("Candidate must be at least 40 years old");
        }
    }

    public String getName() {
        return name;
    }
    public int getCandidateAge(){
        return candidateAge;
    }

    public int getCandidateID() {
        return candidateID;
    }

    public int getVoteCount() {
        return voteCount;
    }

    public void incrementVote() {
        this.voteCount++;
    }

    public void validateCandidatePincode(String pincode) {
        if (!pincode.equals(this.pincode)) {
            throw new IllegalArgumentException("Invalid Pincode");
        }
    }
}
